clusterloader2/testing/network/config.yaml

```diff
@@ -0,0 +1,33 @@
+{{$PROTOCOL := .PROTOCOL}}
+{{$NUMBER_OF_SERVERS := .NUMBER_OF_SERVERS}}
+{{$NUMBER_OF_CLIENTS := .NUMBER_OF_CLIENTS}}
+
+name: network
+automanagedNamespaces: 1
+tuningSets:
+- name: Sequence
```

Comment:
This isn't used anywhere - please remove tuningSets completely.

Reply:
yes modified.